nodepool/nl02.sjc1.vexxhost.zuul.ansible.com.yaml

```diff
@@ -117,6 +119,9 @@ providers:
       - name: vyos-1.1.8-20190407
         config-drive: true
         connection-type: network_cli
+      - name: vqfx-18.1R3-S2.5-20200116
+        config-drive: false
```

Comment:
Why no config-drive? Does image not boot properly

Reply:
I tried to have to config sent there with config-drive and it just didn't work (unlike with vsrx)